code/modules/admin/sql_notes.dm

```diff
@@ -1,4 +1,4 @@
-/proc/add_note(target_ckey, notetext, timestamp, adminckey, logged = 1, checkrights = 1, show_after = TRUE, automated = FALSE, sanitise_html = TRUE) // Dont you EVER disable this last param unless you know what you're doing
```

Comment:
Edit comment to say automated.

Reply:
Comment changed to `// Dont you EVER disable 'sanitise_html', only automated systems should use this`